dotcom-rendering/src/types/front.ts

```diff
@@ -217,7 +232,7 @@ export type FEFrontCard = {
 	card: {
 		id: string;
 		cardStyle: {
-			type: string;
+			type: FEFrontCardStyle;
```

Comment:
So this property is _always_ present, but the one below _may be missing_? What a curious choice…

Reply:
I was confused by this. I think [`PressedContent` in `facia-press`](https://github.com/guardian/frontend/blob/main/common/app/model/pressedContent.scala#L74) could do with some refactoring. Some observations:

1. All types of `PressedContent` (`CuratedContent`, `SupportingCuratedContent`, `LinkSnap` and `LatestSnap`) have a `PressedCard` and this [always has `CardStyle`](https://github.com/guardian/frontend/blob/main/common/app/model/PressedCard.scala#L13).
2. `CuratedContent`, `SupportingCuratedContent` also have the `CardStyle` as a property, hence it becomes optional on this level because it's missing from `LinkSnap` and `LatestSnap`